src/locales/default.json

```
@@ -349,6 +349,7 @@
       "information": "The minimum amount of voting power required to create a proposal"
     },
     "allowOnlyAuthors": "Allow only authors to submit a proposal",
+    "enableAliased": "Allow voting without additional signature",
```

This might be confusing as there is still a signature to secure an action. I'd formulate it like this:
```suggestion
    "enableAliased": "Allow voting with session keys",
```

---

:ok_hand: 

By the way, how do you handle propagating translations?

---

We use Crowdin for translation, see https://translate.snapshot.org